lib/index.ts

```diff
@@ -18,30 +18,28 @@ export { inspect, dockerFile };
 const MAX_MANIFEST_FILES = 5;
 const debug = Debug("snyk");
 
-function inspect(
+async function inspect(
   root: string,
   targetFile?: string,
   options?: any,
 ): Promise<PluginResponse> {
   const targetImage = root;
 
+  const dockerfileAnalysis = await dockerFile.readDockerfileAndAnalyse(targetFile);
+
   if (options && options.experimental) {
-    return experimentalAnalysis(targetImage, options);
+    return experimentalAnalysis(targetImage, dockerfileAnalysis, options);
```

Comment:
how do you feel about adding an ```await``` for each ```return``` statement here?

is idea is having a stricter standards, even when unnecessary, so that future changes will be a bit safer to make (let's say tomorrow someone wants to wrap one of the calls in try/catch clauses and they forget adding ```await``` etc.)
more light reading I just enjoyed https://jakearchibald.com/2017/await-vs-return-vs-return-await/